### ISSUE TITLE
How to implement from SYNTHIA to Cityscapes?

### ISSUE THREAD

**Issue by feipanir** [valeoai/ADVENT#12]
Hello, I am re implementing the adaptation from SYNTHIA to Cityscapes. Except for the image size [1280, 760], I used the exact same setting up with GTA5->Cityscapes, including init parameters, learning rate, iterations, ect.  But according to my training result, i could only get the best mIOU: 39.1% for 16 classes. I would like to ask, how should I improve to get the similar performance as in your paper. It is shown as below:
![image](https://user-images.githubusercontent.com/18341349/67409056-facbdb00-f5f4-11e9-8f0c-9553ec19b50c.png)

**Comment by tuanhungvu** [comment]
Hello @feipan664 , for the training on synthia, we keep most hyper-params used in the GTA5 experiment. The only difference is that we set MAX_ITERS = 90000.
Best,
T-H

**Comment by feipanir** [comment]
Thanks so much for your reply!

**Comment by tuanhungvu** [comment]
Please check this repo for synthia->cityscapes experiments [https://github.com/valeoai/DADA](url)
Solved #12
